### ISSUE TITLE
No icon in activity bar

### ISSUE THREAD

**Issue by wuffo** [moshfeu/vscode-compare-folders#8]
Saw the icon when I first installed it. Tried to select a folder, but it seemed to get stuck. Since then no icon / functionality. Tried to disable / uninstall a few times, but still no icon. Restarted VSC too.

VSC 1.40.1
Ubuntu 18.04

**Comment by moshfeu** [comment]
Thank you! I'll check this soon.

**Comment by JonathanTroyer** [comment]
I have the same issue, with extension info:
```
Name: Compare Folders
Id: moshfeu.compare-folders
Description: Compare folders by contents, present the files that have differences and display the diffs side by side
Version: 0.2.0
Publisher: MoshFeu
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=moshfeu.compare-folders
```

**Comment by moshfeu** [comment]
@JonathanTroyer this is a great feedback. I'll check this.

**Comment by moshfeu** [comment]
First. The activity bar is shows up only when there is only one workspace.

https://github.com/moshfeu/vscode-compare-folders/blob/80bb718fb6202960ecd225fc01567aa0345c2a9b/package.json#L105

I wasn't aware the need of multiple workspaces 😕
First thing, I'll update the README so people will know what to expect.
Then, I'll figure it out how to support multiple workspaces.

**Comment by moshfeu** [comment]
I just published a new version (v0.3.0) that supports multiple workspaces (so the icon will shows up).
Can you verify please?
Thank you for your help!

**Comment by JonathanTroyer** [comment]
So, I get the icon now, but I cannot compare folders.

### To reproduce,
1. Create a folder (I named it folder) and add content (I added test.txt with contents "1").
2. Duplicate that folder, rename it (I named it folder2) and change the content (changed test.txt to contain "2")
3. Either
  3.1. Open the first folder directly
  3.2. Open a blank workspace, add the folder to it, and save
4. Navigate to the compare pane (which is now possible if you do 3.2., thanks!!)
5. Attempt to compare a folder. Whenever I open a folder (folder2), nothing happens

### Current extension information:
```
Name: Compare Folders
Id: moshfeu.compare-folders
Description: Compare folders by contents, present the files that have differences and display the diffs side by side
Version: 0.3.0
Publisher: MoshFeu
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=moshfeu.compare-folders
```

I'm on Windows 8.1 Pro, VSCode 1.41.0 if that helps.

**Comment by moshfeu** [comment]
Thanks @JonathanTroyer. I tried to reproduce, here is the result. What am I missing?


![video1](https://user-images.githubusercontent.com/3723951/71130878-b8410c00-21fb-11ea-85c8-994440f17782.gif)

**Comment by moshfeu** [comment]
Hi again @JonathanTroyer and @wuffo.
Just published v0.4.0 can you check if it's still happening?
Thanks!

**Comment by moshfeu** [comment]
I'm closing it for now.
If anyone get this problem too, I'll be very happy if you could open a new issue
